Beta release does not compile [Windows 10]

the crates.io version ``0.1.0-beta`` does not compile:
```rust
error[E0599]: no method named `expect` found for type `std::result::Result<(), winit::event_loop::EventLoopClosed<Message>>` in the current scope
   --> C:\Users\songtronix\.cargo\registry\src\github.com-1ecc6299db9ec823\iced_winit-0.1.0-alpha\src\application.rs:377:18
    |
377 |                 .expect("Send command result to event loop");
    |                  ^^^^^^ method not found in `std::result::Result<(), winit::event_loop::EventLoopClosed<Message>>`    |
    = note: the method `expect` exists but the following trait bounds were not satisfied:
            `winit::event_loop::EventLoopClosed<Message> : std::fmt::Debug`

error: aborting due to previous error
```

However current master does compile.

--- a/winit/Cargo.toml
+++ b/winit/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "iced_winit"
-version = "0.1.0-alpha"
+version = "0.1.0-alpha.1"
 authors = ["Héctor Ramón Jiménez <hector0193@gmail.com>"]
 edition = "2018"
 description = "A winit runtime for Iced"
@@ -11,7 +11,7 @@ repository = "https://github.com/hecrj/iced"
 debug = []
 
 [dependencies]
-iced_native = { version = "0.1.0-alpha", path = "../native" }
-winit = "0.20.0-alpha4"
+iced_native = { version = "0.1.0", path = "../native" }
+winit = "=0.20.0-alpha4"
 futures = { version = "0.3", features = ["thread-pool"] }
 log = "0.4"

